server/debug.go

```diff
@@ -29,6 +30,37 @@ func (s *Server) registerDebugRoutes(g *echo.Group) {
 
 		return currentDebugState(c)
 	})
+
+	g.GET("/debug/log", func(c echo.Context) error {
+		var errorRecordList []*api.DebugLog
+		// incrementID is used as primary key in jsonapi.
+		var incrementID int
+
+		s.errorRecordRing.RWMutex.RLock()
+		defer s.errorRecordRing.RWMutex.RUnlock()
+
+		s.errorRecordRing.Ring.Do(func(p interface{}) {
+			if p == nil {
+				return
+			}
+			errRecord, ok := p.(*api.ErrorRecord)
+			if !ok {
+				return
+			}
+			errorRecordList = append(errorRecordList, &api.DebugLog{
+				ID:          incrementID,
+				ErrorRecord: *errRecord,
+			})
+			incrementID++
```

Comment:
note: as previously stated, I moved incrementID below so that the list starts with zero.